css/home.css

```diff
@@ -87,12 +87,24 @@
 }
 
 .homepage-hero__lookdown {
+  bottom: 20px;
+  right: 0;
+  left: 0;
+  width: 34px;
+  height: 40px;
+  position: absolute;
+  margin: auto;
+}
+
+.homepage-hero-scroll__lookdown{
```

Comment:
Please add a space between	**.homepage-hero-scroll__lookdown** and **{**

Reply:
done